From 72756297018dabe8db73e5b91db4a90384145fa0 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Thu, 14 Nov 2024 14:37:18 +0100
Subject: [PATCH 1399/1695] dts: stm32: remove provider-cells properties for
 stm32mp23x

Removes theses properties:
- #power-domain-cells
- #clock-cells
- #reset-cells

from scmi node, as theses nodes are not resources provider as same as
their equivalent on the Linux side.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 38ebee1e01140dcd63cb29670cbbfed88bebe8fa dts: stm32: remove provider-cells properties for stm32mp23x]
Fixes: 4df4065b8d ("dts: stm32: describe scmi resources for stm32mp23x")
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ibdcfa35120554b641ee529668e423010099afa9e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/418589
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp231.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 2df192140..ee6ca6707 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -155,7 +155,6 @@
 				power-domains {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#power-domain-cells = <1>;
 
 					pd_gpu: pd@0 {
 						reg = <PD_SCMI_GPU>;
@@ -172,7 +171,6 @@
 				clocks {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#clock-cells = <1>;
 
 					clock@0 {
 						reg = <CK_SCMI_ICN_HS_MCU>;
@@ -1075,7 +1073,6 @@
 				resets {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#reset-cells = <1>;
 
 					reset@0 {
 						reg = <RST_SCMI_C1_R>;
@@ -1184,7 +1181,6 @@
 				clocks {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#clock-cells = <1>;
 
 					clock@0 {
 						reg = <CK_SCMI_ICN_HS_MCU>;
@@ -2088,7 +2084,6 @@
 				resets {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#reset-cells = <1>;
 
 					reset@0 {
 						reg = <RST_SCMI_C1_R>;
-- 
2.47.2

